src/styles.css

```diff
@@ -0,0 +1,159 @@
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap");
+.App {
+  font-family: sans-serif;
+}
+
+.container{
+  margin: auto;
+  width: 600px;
+  height: 400px;
+  position: relative;
+  text-align: center;
+  padding: 15px;
+  box-shadow: 0 2px 30px rgba(255, 255, 255, 0);
+  color: rgb(219, 139, 163);
+  
+}
+
+.clock {
+  width: 300px;
+  height: 300px;
```

Comment:
It would be wiser to use relative units here as well. Try to find a way to make it proportional while being responsive, even if it means using JS.
Test it with the responsiveness feature in the dev tools.
(This icon <img src="https://developer-chrome-com.imgix.net/image/admin/9FiBHFCzfPgP8sy6LMx7.png?w=12"> in the F12 menu)